### DISCUSSION TITLE
Some general ideas/links to ideas on how to support project maintainers

### DISCUSSION THREAD

**Discussion by starksm64** [commonhaus/foundation#3] [Ideas]
One of my big concerns is how project maintainers might be supported. This discussion starts out with just a list of existing ideas I have found. They need to be digested, critiqued and evolved into an actionable plan.

- [8 ways your company can support and sustain open source](https://opensource.com/article/19/4/ways-support-sustain-open-source)
- [Sustaining Maintaining](https://daverupert.com/2021/12/sustaining-maintaining/)
- [Open Source Needs Maintainers. But How Can They Get Paid?](https://thenewstack.io/open-source-needs-maintainers-but-how-can-they-get-paid/)
- [How Open Source Software Maintainers Survive](https://www.zentao.pm/open-source-community/open-source-project-maintainer-survivals-292.html)
- [Ensuring open source sustainability](https://opensenselabs.com/blog/articles/open-source-sustainability)
- [The burden of an Open Source maintainer](https://www.jeffgeerling.com/blog/2022/burden-open-source-maintainer)
- [How To Incentivize Open Source Maintainers for More Security](https://www.spiceworks.com/tech/devops/guest-article/how-to-incentivize-open-source-maintainers-for-more-security/)
- [Who will maintain the maintainers?](https://www.runtime.news/who-will-maintain-the-maintainers-open-source-groups-push-for-a-sustainable-future/)
- [Open Collective](https://opencollective.com)
- [Open Source Collective](https://www.oscollective.org)
- [Software in the Public Interest (SPI)](https://www.spi-inc.org)
- [GitHub Sponsors](https://github.com/sponsors)

**Comment by ebullient** [comment]
Yes. Big questions here. OpenCollective/Open source collective are much more focused on how to support maintainers. This is part of their charter and reason for existing. SPI can do this as well, but SPI is oriented around single projects, where OpenCollective is built to support other collectives.

**Comment by starksm64** [comment]
Ok, added those to the list as well as GitHub sponsors.

One thought I have is to have projects run events including crowd source funding events for releases and even CVEs. This requires a platform that supports the effort, and it would be good if it could easily enable print on demand rewards for said events.

**Reply by aalmiray** [reply]
Here's another idea: run hackathon/hacking events while accepting donations for a given project/cause. This is what GDQ (Games Done Quick) do, where gamers play games and donations are usually posted to non-profit orgs and cure for cancer research.
Additional donations let donors chose what a gamer should or must do as a challenge. 

In the case of FLOSS donations could be sent to specific projects, challenges could be "fix bug XYZ" or "help project X with Y" and such. The pool of possible options should be decided before hand and agreed with the projects that could receive a contribution. That kind of disarms a "pay to play" situation. 

There are other things that should be discussed snd considered before bringing this idea to life but I thought it would be good to explore where this could go.

**Comment by ShaneCurcuru** [comment]
There are plenty of sites with single stories or lists of sustainability-related topics, which is great, and expanding nicely.  But we need more people working on different aspects of sustainability (like Commonhaus!), especially in terms of crafting messages that can be used effectively for different audiences: maintainers, CTOs/OSPOs, CEOs, government policy makers, university CS departments, etc.  

I'm trying to collect key links here, especially listing the different aspects of sustainability that need solutions - like foundations!

https://fosssustainability.com/aspects

**Comment by aalmiray** [comment]
Here's another option https://www.opentech.fund/funds/free-and-open-source-software-sustainability-fund/

**Comment by sigrunixia** [comment]
This was brought to my attention today as something complementary to our org. 

https://blog.tidelift.com/paying-maintainers-the-howto

The social thread has some good caveats as well.

https://social.coop/@luis_in_brief/112281757671200311

**Comment by ebullient** [comment]
GitHub has a community for maintainers to support each other: https://maintainers.github.com/
